src/user_default.erl

```diff
@@ -0,0 +1,14 @@
+-module(user_default).
+
+-export([c/1]).
+
+c(File) ->
+    % TODO: we can be a lot more clever about recognizing
+    % if the argument is a file or a module, just like
+    % shell_default:c/1.
+    case gradualizer:type_check_file(File) of
```

Comment:
We could skip gradualizing if File is an atom. Then, `c(my_module)` is still usable at least.

Reply:
Yes.